partcad/src/partcad/runtime_python_conda.py

```diff
@@ -61,7 +62,7 @@ def run(self, cmd, stdin=""):
                     "--no-capture-output",
                     "-p",
                     self.path,
-                    "python",
+                    self.python,
```

Comment:
It is not clear why this needs to be touched in the context of this issue. It seems like an important topic that we need to discuss and track as a separate project/issue. My first impression is that it is a wrong change as it enforces the use of a particular Python executable from the host environment instead of letting conda to find the appropriate one within the conda environment created for whatever Python version is required in that particular package.

Reply:
If we are leaving "python" as argument then cadquery should be installed in the base conda environent. Otherwise it will throw errors (No module found). Usually users will create new  conda environment  for deal kind of project. In that case it may fail. I faced this issue and fixed by `self.python=shutil.which(python)`.